docs/team/alekkwek18.md

```diff
@@ -31,8 +26,11 @@ Given below are my contributions to the project.
 
 * **Enhancements to existing features**:
   * Refactored `AddCommand` to `AddContactCommand` to improve specificity and clarity.
-  * Refactored `DeleteCommand` into a generic command `DeleteCommand<T>` to support future implementations of `DeleteJob` and `DeleteCompany`.
+  * Refactored the original functionality of `DeleteCommand` into `DeleteContactCommand` to allow the user to delete a specific contact from the address book.
+  * Refactored `DeleteCommand` into a generic command `DeleteCommand<T>` to support future implementations of `DeleteContactCommand`,`DeleteJobCommand` and `DeleteCompanyCommand`.
+  * Refactored `Person` class from tags to skills.
```

Comment:
Now that I think about it, not sure if there are features since it is just an internal change, but let just keep it.